collectors/o365/o365_collector.js

```diff
@@ -83,6 +83,16 @@ class O365Collector extends PawsCollector {
 
     pawsGetLogs(state, callback) {
         let collector = this;
+
+        if (!moment(state.since).isValid() || !moment(state.until).isValid() || state.since === undefined || state.until === undefined) {
+            const newStartTs = moment();
+            state.since = newStartTs.toISOString();
+            state.until = newStartTs.add(collector.pollInterval, 'seconds').toISOString();
+            state.nextPage = null;
+            state.poll_interval_sec = collector.pollInterval;
```

Comment:
I would just use the built in progressions that we use elsewhere in the collector.

https://github.com/alertlogic/paws-collector/blob/master/collectors/o365/o365_collector.js#L165

Reply:
Done